.github/workflows/ci-vulnerability-scans.yml

```diff
@@ -79,7 +96,15 @@ jobs:
         uses: anchore/scan-action@v3
         with:
           image: ${{ steps.build-image.outputs.image }}
-          output-format: "table"
+          output-format: "json"
+
+      - name: Save output to workflow summary
+        if: always() # Runs even if there is a failure
+        run: |
+          echo "# Anchore Results" >> $GITHUB_STEP_SUMMARY
+          if [ -f results.json ]; then
+            cat results.json | jq --raw-output -f .anchore_format.jq >> $GITHUB_STEP_SUMMARY
+          fi
```

Comment:
It's annoying that [anchor scan](https://github.com/anchore/scan-action) doesn't allow us to write to a file when using "table" format.

To keep things simple, I'm leaning towards just outputting the anchor scan results as a table, and then having the summary have instructions to tell the developer to look at the logs
```suggestion
          output-format: table

      - name: Workflow summary
        if: always() # Runs even if there is a failure
        run: |
          echo "# Anchore Results" >> $GITHUB_STEP_SUMMARY
          echo "See GitHub Action logs for anchor scan results"
```